dangvuthang/rocodingtest
# Setup frontend app with NextJs, MaterialUI

### Changelog
Setup frontend app with `NextJs`, and `MaterialUI`
### How to test

1. Clone this repo
2. `cd` to `frontend` folder
3. run `npm install`
4. run  `npm run dev` to see if it works correctly
